1.1/geo.ttl

```diff
@@ -119,12 +119,22 @@ geosparql:
 :dggsLiteral
 	a rdfs:Datatype ;
 	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
-	skos:definition """A textual serialization of a Discrete Global Grid (DGGS) geometry object."""@en ;
-	skos:example """ "<https://w3id.org/dggs/auspix> OrdinateList (R3234)"^^<http://www.opengis.net/ont/geosparql#dggsLiteral>""" ;
-  rdfs:seeAlso <http://www.opengis.net/doc/AS/dggs/2.0> ;
+	skos:definition """A textual serialization of a Discrete Global Grid System (DGGS) geometry object."""@en ;
+	skos:scopeNote "This datatype is not expected to be used directly but to serve as an abstract datatype for a series of specific DGGS literal types, for specific DGGS implementations" ;
 	skos:prefLabel "DGGS Literal"@en ;
 .
 
+:auspixDggsLiteral
+	a rdfs:Datatype ;
+	rdfs:subClassOf :dggsLiteral ;
+	rdfs:isDefinedBy geosparql: , geosparql-spec: ;
+	rdfs:seeAlso "https://github.com/GeoscienceAustralia/AusPIX_DGGS"^^xsd:anyURI ;
+	skos:definition """A textual serialization of an AusPix Discrete Global Grid System (DGGS) geometry object."""@en ;
+	skos:example """ "OrdinateList (R3234)"^^geo:auspixDggsLiteral""" ;
+	skos:scopeNote "This datatype is to be used only for a specific DGGS implementation - AusPix. Other DGGS implementations should declare their own datatypes."	
+	skos:prefLabel "AusPIX DGGS Literal"@en ;
+.
```

Comment:
I wonder if the main GeoSPARQL ontology is the proper place to define different flavours of dggsLiteral. Other/future DGGS implementations won't have the luxury to add their specification to GeoSPARQL, I presume.

Reply:
True, hence my persistent cry for supplementary vocabs for ontologies! Here though, we can have such a serializations vocab but the ontology benefits from at least one DGGS implementation to allow for real examples in documentation. the long-term home for all serializations though should be a vocab that we've not yet made but will need to supply for 1.1+ since there are calls for many more literal forms.

Reply:
@nicholascar: did you mean to write "supplementary vocabs for DGGS serialisations"? Otherwise I don't understand that part of your comment.

Perhaps this is not the place to discuss whether ` :auspixDggsLiteral` should be in GeoSPARQL. I see it has already made it to the master branch. Is the only reason to have ` :auspixDggsLiteral` in the GeoSPARQL ontology to make some of the examples work? Couldn't the examples make use of the `ex` namespace? 

Perhaps the DGGS situation is comparable to the CRS situation. For CRSs there is clear need for a standard CRS ontology and a accompanying CRS registry. But shouldn't those be projects separate from GeoSPARQL?